src/css.js

```diff
@@ -165,6 +165,10 @@ function getWidthOrHeight( elem, name, extra ) {
 	) + "px";
 }
 
+function isCustomProperty( value ) {
+	return value.match( /--(.*)/ );
```

Comment:
A few remarks:

1. This will match things like `a--b`; you should check that the string starts with `--`, not just contains it.
1. Why did you introduce the group `(.*)`? You don't use it anywhere.
1. It's better to change that to `/^--.*`/.test( value )` as you only care about it matching or not and what you wrote will return the array with matches; the browser has better chances of optimising the former.